spec/environment_helpers/file_helpers_spec.rb

```diff
@@ -0,0 +1,72 @@
+RSpec.describe EnvironmentHelpers::FileHelpers do
+  subject(:env) { ENV }
+
+  describe "#file_path" do
+    let(:name) { "FOO" }
+
+    context "with required: true" do
+      subject(:file_path) { ENV.file_path(name, required: true) }
+
+      context "when the environment value is not set" do
+        before { expect(ENV["FOO"]).to be_nil }
+
+        it "raises a MissingVariableError" do
+          expect { file_path }.to raise_error(EnvironmentHelpers::MissingVariableError, /not supplied/)
+        end
+      end
+
+      context "when the environment value is set" do
+        context "but it does not point to an existing file" do
```

Comment:
In the other methods, I've taken `required: true` generally to mean "raise an error if something goes wrong" - I would have expected to see an `EnvironmentHelpers::FileNotFound` or something like that for this case, I think?

Might be better to use a file that _definitely_ doesn't exist, as well, like `__FILE__ + ".notarealfile"`

Reply:
I think we should allow for a handle to point to a record that does _not yet exist_. It could be an _output_ file or something. I've updated it to return a Pathname object pointing to the ENV variable, regardless of existence.